src/main/kotlin/apply/ui/admin/mail/MailPreviewDialog.kt

```diff
@@ -0,0 +1,53 @@
+package apply.ui.admin.mail
+
+import com.vaadin.flow.component.Component
+import com.vaadin.flow.component.Html
+import com.vaadin.flow.component.button.Button
+import com.vaadin.flow.component.dialog.Dialog
+import com.vaadin.flow.component.html.H2
+import com.vaadin.flow.component.orderedlayout.FlexComponent
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout
+import com.vaadin.flow.component.orderedlayout.VerticalLayout
+import org.jsoup.Jsoup
+import support.views.createContrastButton
+
+class MailPreviewDialog(
+    htmlString: String
+) : Dialog() {
+    init {
+        add(createHeader(), createHtmlComponentFrom(htmlString), createButtons())
+        width = "700px"
+        height = "800px"
+        open()
+    }
+
+    private fun createHeader(): VerticalLayout {
+        return VerticalLayout(H2("메일 미리보기")).apply {
+            isPadding = false
+            element.style.set("margin-bottom", "10px")
```

Comment:
```suggestion
            element.style["margin-bottom"] = "10px"
```

린트가 이렇게 바꾸면 더 좋다고 하네요

가독성 및 이해도
함수 호출의 경우 독자가 호출된 함수가 무엇을 하는지 알아야 하는 반면, 인덱싱된 액세스 연산자의 의미는 독자에게 명확하기 때문에 이 변경으로 함수가 무엇을 하는지 더 쉽게 이해할 수 있습니다.

Reply:
operator 사용이 아직 낯서네요,,,
어드민 다른 코드들에 `style.set()` 을 사용하는 경우가 많은데 이것들도 바꿔야겠네요 ㅎㅎ